File: app/src/main/java/com/simplemobiletools/draw/PaintOptions.java
package com.simplemobiletools.draw;

import android.graphics.Color;

class PaintOptions {
    int color = Color.BLACK;
    float strokeWidth = 5f;

    PaintOptions() {

    }

    PaintOptions(int color, float strokeWidth) {
        this.color = color;
        this.strokeWidth = strokeWidth;
    }
}

File: app/src/main/kotlin/com/simplemobiletools/draw/PaintOptions.kt
package com.simplemobiletools.draw

import android.graphics.Color

class PaintOptions {
    var color = Color.BLACK
    var strokeWidth = 5f
    var isEraser = false

    constructor()

    constructor(color: Int, strokeWidth: Float, isEraser: Boolean) {
        this.color = color
        this.strokeWidth = strokeWidth
        this.isEraser = isEraser
    }

    fun getColorToExport() = if (isEraser) "none" else Integer.toHexString(color).substring(2)
}
